src/script/sign.cpp

```diff
@@ -121,9 +121,6 @@ static bool SignStep(const SigningProvider& provider, const BaseSignatureCreator
     {
```

Comment:
It seems that there is still some processing related to scriptWitness.
Remove `witness_script` and `scriptWitness` from SignatureData in `script/sign.h` to clarify the target.